test_conformance/run_conformance.py

```diff
@@ -63,19 +66,33 @@ def get_tests(filename, devices_to_test):
         comment = re.search("^#.*", line)
         if comment:
             continue
-        device_specific_match = re.search("^\s*(.+?)\s*,\s*(.+?)\s*,\s*(.+?)\s*$", line)
+        if sys.version_info[0] < 3:
+            device_specific_match = re.search("^\s*(.+?)\s*,\s*(.+?)\s*,\s*(.+?)\s*$", line)
+        else:
+            device_specific_match = re.search("^\s*(.+)\s*,\s*(.+)\s*,\s*(.+)\s*", line)
         if device_specific_match:
             if device_specific_match.group(1) in devices_to_test:
-                test_path = string.replace(device_specific_match.group(3), '/', os.sep)
-                test_name = string.replace(device_specific_match.group(2), '/', os.sep)
+                if sys.version_info[0] < 3:
+                    test_path = string.replace(device_specific_match.group(3), '/', os.sep)
+                    test_name = string.replace(device_specific_match.group(2), '/', os.sep)
```

Comment:
str.replace seems to work fine with Python 2.  Is there a reason not to change this for Python 2 as well, to use str.replace instead of string.replace?  (If there is not, then the `sys.version_info[0] < 3` check can be removed.)

Reply:
Again, I did not make the changes, if it works fine then it should be okay to remove it. I think str replace and string.replace is analogous in python 2.

Reply:
Please use only `str.replace`, if it works with both Python 2 and Python 3.

Reply:
Now done.